Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,241 @@
+Obtaining .NET CLI 
+==================
+
+# Contents
+* [Overview](#overview)
+* [General principles](#general-principles)
+* [Channels](#channels)
+* [Dependencies](#dependencies)
+* [Upgrades](#upgrades)
+* [Layout on disk](#layout-on-disk)
+* [Acquisition vectors](#acquisition-vectors)
+  * [Getting Started page](#getting-started-page)
+  * [Repo landing page](#repo-landing-page)
+* [Acquisition modes](#acquisition-modes)
+  * [Native installers](#native-installers)
+  * [Installation script](#installation-script)
+    * [Windows one-liner](#windows-command)
+    * [OSX/Linux one-liner](#osxlinux-shell-command)
+  * [Complete manual installation](#complete-manual-installation)
+  * [Docker](#docker)
+  * [NuGet Packages](#nuget-packages)
+* [Acquiring through other products](#acquiring-through-other-products)
+  * [Visual Studio](#visual-studio)
+  * [VS Code](#vs-code)
+  
+
+# Overview
+This document/spec outlines the CLI install experience. This covers the technologies being used for install, the principles that are driving the installation experience, the ways users are coming to the installs and what each of the installs contains, in terms of stability and similar. 
+
+# General principles
+
+- All installers are signed properly 
+- Upgrades using the native installers Just Work(tm)
+- All user facing materials point to the getting started page
+- The user needs extra effort to install the "bleeding edge" bits
```

Comment:
"bleeding edge" is not clear for everyone what this means - it should probably be "nightly build"